src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html

```diff
@@ -14,7 +14,7 @@
     <ng-template #elsePods>
       <div>
         Pod status:
-        <li *ngFor="let column of pods.pods">
+        <li *ngFor="let column of pods.pods"  [attr.id]="'pod_status_' + column[0]">
```

Comment:
What does `[attr.id]` do? Why not just use `id`?

Reply:
I've used this answer in StackOverflow 

https://stackoverflow.com/questions/34568497/how-to-set-dynamic-idngfor-in-ionic2-angular2

I've just tried id = "'pod_status_' + column[0]" and it did not work.

Reply:
good to know, thanks